src/Pages/Main/BestSeller.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from "styled-components";
+import Item from "./Item";
+
+function BestSeller() {
+  return (
+    <Best>
+      <div className="nameWrapper">
+        <h2 className="name"> BEST SELLER</h2>
+      </div>
+      <div className="videoWrapper">
+        <video muted loop autoPlay>
+          <source
+            src="https://www.laneige.com/kr/ko/assets/video/index/bestseller-bg.mp4"
+            type="video/mp4"
+          />
+        </video>
+      </div>
+      <Item />
+    </Best>
+  );
+}
+
+const Best = styled.section`
+  width: 100%;
+  position: relative;
+  .nameWrapper {
+    position: absolute;
+    width: 100%;
+    top: 50px;
+    text-align: center;
+  }
+  .name {
+    font-size: 42px;
+    font-weight: bold;
+  }
+  .videoWrapper {
+    width: 100% !important;
+    overflow: hidden;
+    height: 460px;
+  }
+  .video {
+    width: 100% !important;
+    object-fit: fill;
+  }
```

Comment:
```suggestion
    font-weight: bold;
  }
  
  .videoWrapper {
    width: 100%;
    overflow: hidden;
    height: 460px;
  }
  
  .video {
    width: 100%;
    object-fit: fill;
  }
```
- 셀렉터 간 한줄 씩 띄워주세요
- important 지속적으로 사용하고 계신데 꼭 필요할 때만 쓰셔야합니다!
- class 셀렉터 사용 가능하긴 한데 그냥 스타일드 컴포넌트로 사용하시는 건 어떨까요?